process/project-lifecycle.md

```diff
@@ -1,156 +1,168 @@
-* Have a healthy number of maintainers. A maintainer is defined as someone with the commit bit; i.e., someone who can accept and merge contributions to some or all of the project.
-* Demonstrate a substantial ongoing flow of commits and merged contributions.
-* Since these metrics can vary significantly depending on the type, scope and size of a project, the TAC has final judgement over the level of activity that is adequate to meet these criteria
-* A clear versioning scheme.
-* Specifications must have at least one public reference implementation.
+#### Project Process: Sandbox to Incubating
 
-#### Project Graduation Process: Incubating to Graduation
+Incubating Projects are required to undergo technical due diligence as a part of the process to move from Sandbox to Incubation. Technical Due Diligence is driven by a TAC or parent WG sponsor. Once the diligence is complete and the proposal made, two weeks are allocated for public comment before a TAC vote is called.
 
-Projects that wish to move from Incubating to Graduation should open a PR confirming the following criteria:
-* Have maintainers from at least two organizations.
-* Have achieved and maintained an [OpenSSF Best Practices Badge](https://bestpractices.coreinfrastructure.org/).
-* Have completed an independent and third party security audit with results published of similar scope and quality as [this example](https://github.com/envoyproxy/envoy#security-audit) which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation. The TAC, or another WG, may provide assistance in acquiring such an audit.
-* Explicitly define project governance and process to be granted maintainership within the project team. This is preferably laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus maintainers.
-* Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website). For a specification, have a list of adopters for the implementation(s) of the spec.
-* Please include a short one-page narrative based off the Graduation template, no more than 500 words.
 
 #### Submission Forms for Incubating
 
 1. [Proposal](TBD)
 2. [Legal](TBD)
 
-#### Submission Forms for Incubating to Graduation
-
-1. [Proposal](TBD)
-2. [Legal](TBD)
 
 ### Graduated
 
-Graduated projects signal the highest level of maturity for an OpenSSF project.
+Graduated projects signal the highest level of maturity for an OpenSSF project. Graduated projects have consistent, mature open source development practices with sustained contribution and activity. 
+
+#### Project Reponsibilities
+
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
```

Comment:
Is it worth being more explicit here in that we want them to achieve Passing or Silver badging criteria before they can graduate?

Reply:
Probably depends on if we want to "hardcode" to the CII badging program, or if "as recommended by the OpenSSF" implies that we expect some level of maturity to be demonstrated through OpenSSF programs (which could include CII badges). 

I'm more inclined to the later and to change to the former if the TAC feels like they're receiving graduation submissions that don't follow best practices, and having a certain badge level would have addressed that.

Reply:
The OSSF Badge project is a vetted set of good security practices.  I feel strongly that all foundation projects should "dogfood" our own practices and get a badge to send the message we practice what we preach.

Reply:
We should be continually evolving our security expectations as a security foundation.  To that end these badges have been out for a while, silver in my mind is a minimum bar.

Reply:
@AevaOnline @bobcallaway @inferno-chromium @SecurityCRob -- any +1s on Silver?

Reply:
+1 to Silver, esp since it brings in the security testing requirements. Our own security tool need to pass the basic security criteria we preach to others (as @SecurityCRob said)

Reply:
+1 to silver as well.